app/helpers/assignment_index_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AssignmentIndexHelper
+  def cal_day(day:)
+    'current-day' if day == Date.current
+  end
+
+  def cal_assignment(assignment:, day:)
+    return if assignment.blank?
+    ass = 'assignment'
```

Comment:
Okay, I don't care about it being inappropriate. However distracting it is for a programmer to read `ass` over and over again. 

All I care about is how well the name represents what it is/does. I figured that `assignment` would not work out. Ummmmm. I'll think of a good alternative if you can't think of something.